apps/meteor/client/views/meet/CallPage.tsx

```diff
@@ -71,36 +71,34 @@ const CallPage: FC<CallPageProps> = ({
 				}
 				return false;
 			};
-			Notifications.onUser(
-				WEB_RTC_EVENTS.WEB_RTC,
-				(type: any, data: any) => {
-					if (data.room == null) {
-						return;
-					}
-					webrtcInstance.onUserStream(type, data);
-				},
-				visitorId,
-			);
-			Notifications.onRoom(roomId, 'webrtc', (type: any, data: any) => {
+			Notifications.onVisitor(WEB_RTC_EVENTS.WEB_RTC, visitorId, (type: any, data: any) => {
+				if (data.room == null) {
+					return;
+				}
+				webrtcInstance.onUserStream(type, data);
+			});
+			// TODO: I have no idea what this is doing, how this works, why it's here or how to type it
```

Comment:
once this is merged nobody will know who "I" is.. maybe you should sign this comment?

btw, I don't see much relevance of having this comment in the code as well

Reply:
just wanted to make clear why I inserted any instead o typing the stream. I